psalm-baseline.xml

```diff
@@ -176,11 +160,16 @@
     <PropertyNotSetInConstructor occurrences="1">
       <code>$name</code>
     </PropertyNotSetInConstructor>
+    <RedundantCastGivenDocblockType occurrences="4">
+      <code>(bool) $flag</code>
+      <code>(string) $description</code>
+      <code>(string) $name</code>
+      <code>(string) $type</code>
+    </RedundantCastGivenDocblockType>
```

Comment:
These redundant casts cannot be purged as they would result in a BC break, see:

https://github.com/laminas/laminas-server/blob/6488e14a50c7308c2d1aecc19e8305b27e560da2/test/Method/ParameterTest.php#L70-L78

Reply:
You can suppress them individually, or add them to the baseline, then.